### ISSUE TITLE
General web-app appearance

### ISSUE THREAD

**Issue by interrogator** [acdh-oeaw/wugsy#6]
The app is currently functional, but:

* Doesn't have a lot of information (e.g. about pages, links, etc.)
* Could be prettier

This kind of stuff will need to be fixed at some point!

**Comment by notsomes** [comment]
Wouldn't worry a lot about it now, right?

Is the site expected to get online anytime (super)soon? If not, I'd leave that for the later stages when we already have stuff going and available even if it's not fully finished.

**Comment by interrogator** [comment]
Nah, it's a low priority, of course. But, when someone is struggling to find something to do, making something look nicer is better than nothing 👍
